src/libraries/System.Runtime.InteropServices.JavaScript/gen/JSImportGenerator/Diagnostics/Events.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Interop.Diagnostics
+{
+    [EventSource(Name = "Microsoft-Interop-SourceGeneration-Events")]
+    internal sealed class Events : EventSource
```

Comment:
Jeremy or @elinor-fung can confirm this, but I think we removed ours because the source generators now have a reporting mechanism.

Reply:
`src\libraries\System.Runtime.InteropServices\gen\LibraryImportGenerator\Diagnostics\Events.cs` is still there. Could it be removed ?

Reply:
Roslyn now includes ETW events for source generators. We can remove all of ours (the one in LibraryImportGenerator is unused to my knowledge)

Reply:
okay, removed mine thanks!